packages/@aws-cdk/integ-tests-alpha/lib/assertions/providers/provider.ts

```diff
@@ -13,6 +14,13 @@ export interface LambdaFunctionProviderProps {
    * @default index.handler
    */
   readonly handler?: string;
+
+  /**
+   * How long, in days, the log contents will be retained.
+   *
+   * @default - no retention days specified
```

Comment:
should we set some days as default ?

Reply:
Not in this case, since the default behaviour is no log retention. Once users provide any value, we would then add a LogGroup resource.